### PR TITLE
docker-exec: add page

### PR DESCRIPTION
**Pull request by eshack94**
- [x] The page (if new), does not already exist in the repo.
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/docker-exec.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`docker exec -it {{container-name}} /bin/sh`
+
+- Enter an interactive bash pseudo-TTY on an already-running container:
+
+`docker exec -it {{container-name}} /bin/bash`
+
+- Run a command in the background (detached) on a running container:
+
+`docker exec -d {{container-name}} {{command}}`
```
Comment (owenvoke) [inline pages/common/docker-exec.md:16]: Apologies, another underscore here:
```suggestion
`docker exec -d {{container_name}} {{command}}`
```

**Review by owenvoke — CHANGES_REQUESTED** [review]
Hi @eshack94, just a few review comments for using underscores in the tokens, and also possibly merging some of the examples. 🤔 Cheers.

File: pages/common/docker-exec.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# docker exec
+
+> Execute a command on an already running Docker container.
+> More information: <https://docs.docker.com/engine/reference/commandline/exec/>.
+
+- Enter an interactive shell session on an already-running container:
+
+`docker exec -it {{container-name}} /bin/sh`
```
[the remark this answers is not in the document]
Comment (eshack94) [reply pages/common/docker-exec.md:8]: Changes made. Thank you for the feedback!

File: pages/common/docker-exec.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+`docker exec -d {{container-name}} mkdir {{directory-path}}`
+
+- Select the working directory for a given command to execute into:
+
+`docker exec -it -w {{path/to/directory}} {{container-name}} {{command}}`
+
+- Run a command in background on existing container but keep STDIN open:
+
+`docker exec -i -d {{container-name}} {{command}}`
```
[the remark this answers is not in the document]
Comment (eshack94) [reply pages/common/docker-exec.md]: Changes made. Thanks for the feedback! 👍🏼

File: pages/common/docker-exec.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`docker exec -d {{container-name}} {{command}}`
+
+- Create a directory in the background on a running container:
+
+`docker exec -d {{container-name}} mkdir {{directory-path}}`
+
+- Select the working directory for a given command to execute into:
+
+`docker exec -it -w {{path/to/directory}} {{container-name}} {{command}}`
```
[the remark this answers is not in the document]
Comment (eshack94) [reply pages/common/docker-exec.md]: Changes made.

File: pages/common/docker-exec.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+`docker exec -it {{container-name}} /bin/bash`
+
+- Run a command in the background (detached) on a running container:
+
+`docker exec -d {{container-name}} {{command}}`
+
+- Create a directory in the background on a running container:
+
+`docker exec -d {{container-name}} mkdir {{directory-path}}`
```
[the remark this answers is not in the document]
Comment (eshack94) [reply pages/common/docker-exec.md]: Merged into one example. Good feedback. Thank you!

File: pages/common/docker-exec.md (hunk 1 above)
Comment (eshack94) [reply pages/common/docker-exec.md]: Changed based on feedback. Thanks again.

**Review by owenvoke — APPROVED** [review]
Perfect! Thanks @eshack94.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**